workflow/workflow_commands.go

```diff
@@ -343,6 +353,25 @@ func printWorkflowProgress(c *cli.Context, wid, rid string, watch bool) error {
 	}
 }
 
+func printReplayableHistory(iter iterator.Iterator[*historypb.HistoryEvent]) error {
+	var events []*historypb.HistoryEvent
+	for iter.HasNext() {
+		event, err := iter.Next()
+		if err != nil {
+			return err
+
+		}
+		events = append(events, event)
+	}
+
+	history := &historypb.History{}
+	history.Events = events
+
+	common.PrettyPrintJSONObject(history)
```

Comment:
Just verifying that this will not generate pretty output when stdout is not a TTY.